Add support for outbound squid proxy on EC2

djwhatle opened the pull request:
- Add boolean switch "ec2_use_proxy"
   - Deploys an additional EC2 instance with squid proxy configured
   - EC2 instance for squid will forward traffic for any machine on-VPC
   - Sends all outbound OpenShift traffic through squid
   - Configures system level proxy
   - Configures docker proxy
   - Configures oc cluster up proxy
   - Configures broker proxies with "oc patch"

 - Misc changes required for multi-instance (OpenShift + squid) deploy
   - Add support for termination of multiple instances
   - Add support for listing info on multiple instances

 - Add delay before running apb relist during secret filter, seemed to speed up process
 - Change all mentions of "EC-2" to "EC2" to reflect the official name
 - Tweaked various delays to speed up deploy process

johnkim76 commented on ansible/roles/aws_display_info/tasks/minimal.yml:
```diff
@@ -2,17 +2,22 @@
 - debug: var=my_ec2_facts
 
 - block:
+    - debug:
```
Nice, this greatly improves the EC2 instance information!

And could we also show if the proxy option is enabled, and display any information that may of use to the end user?

djwhatle commented:
After looking through this again, it looks like I might have some naming issues given that kubernetes support is now in catasb.

djwhatle commented:
- Added some fixes for secret filtering reliability (should be faster in general now, runs a relist every time)
 - Added support for displaying proxy info

Still need to test this with a 3.7 broker containing the backported proxy fixes. Waiting on an image to be pushed.

johnkim76 commented:
successfully tested with the modified v3.7 broker and AWS APB's in the 'awsserviceborker' org.
